### PR TITLE
Added Vietnamese localization

### PR DESCRIPTION
**Pull request by manhcuong0312**
I've translated the documentation and interface of this add-on into Vietnamese. I also see some misspell in the English documentation. I could fix and contribute the right one, but don't know that I'm allow to do that or not.
Cuong

### CODE REVIEW

**Comment by AAClause** [comment]
Thanks a lot for this contribution!

> I also see some mispell in the English documentation. I could fix and contribute the right one, but don't know that I'm allow to do that or not.

Sure, feel free to suggest these corrections (in another PR). I am certain that @sofquipeut will be happy to receive them. :)

File: addon/doc/vi.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
+﻿---
+author:
+- Sof &lt;<hellosof@gmail.com>&gt;
```
Comment (AAClause) [inline addon/doc/vi.md]: Please replace with your name and your email address (if you want).

File: addon/doc/vi.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+- *NVDA+ALT+n*: chuyển đổi Markdown thành HTML. Kết quả hiển thị trên màn hình ảo của NVDA.
+- *NVDA+ALT+k*: chuyển đổi HTML thành markdown. Kết quả hiển thị trên màn hình ảo của NVDA.
+- *NVDA+ALT+l*: chuyển đổi Markdown thành mã nguồn HTML. Kết quả hiển thị trên màn hình ảo của NVDA.
+- *NVDA+SHIFT+g*: chuyển đổi HTML thành Markdown. Kết quả được chép vào bộ nhớ tạm.
+- *NVDA+SHIFT+h*: chuyển đổi Markdown thành định dạng HTML . kết quả được chép vào bộ nhớ tạm.
+- *NVDA+CTRL+h*: chuyển đổi Markdown thành mã nguồn HTML. Kết quả được chép vào bộ nhớ tạm.
+
+## Thông tin Add-on
+
+* Tác giả:
```
Comment (AAClause) [inline addon/doc/vi.md:231]: Please add an entry for Vietnamese here. Also feel free to update the English documentation for this.

Comment (manhcuong0312) [reply addon/doc/vi.md:231]: Done, and I hope I understand you clearly. I also fix some spelling error that I see.

**Review by AAClause — APPROVED** [review]

**Comment by AAClause** [comment]
Thanks again for this contribution. I've just released a version with your changes!

**Pull request merged**